PHP/PHPhomework1.php

```diff
@@ -0,0 +1,7 @@
+<?php
+$sum = 0;
+for( $i = 1; $i <= 50 ; $i = $i + 2 ){
+  $sum = $sum + $i;
```

Comment:
Indent is 4 spaces, please fix. 
You can setting indentation for your vscode following https://stackoverflow.com/questions/34174207/how-to-change-indentation-in-visual-studio-code